fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -37,8 +37,11 @@
 
 enum class RelocateReason { INVALID = -1, OTHER, REBALANCE_DISK, REBALANCE_READ };
```

Comment:
Reframing it as follows would be cleaner than adding RebalanceOperation

```suggestion
enum class RelocateReason { INVALID = -1, OTHER, REBALANCE_DISK, MOVE_READ_LOAD, SHARE_READ_LOAD };
```

Reply:
Remove `RelocateReason`